instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+1.  **Welcoming to the community.**
+    1.  Once the trainee has passed the demonstration lesson,
+        we generate a PDF certificate for the trainee by running a command-line script in
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
+1.  We keep a list of trainees in a Google Docs spreadsheet with the following fields,
+    which we upate manually:
+    1.  name
+    1.  contact email
+    1.  which training course they took part in
+    1.  the name of the Software Carpentry lesson they're concentrating on (if any)
+    1.  the name of the Data Carpentry lesson they're concentrating on (if any)
+    1.  the URL of their Software Carpentry exercise submission (if any)
+    1.  whether or not they have submitted something for Data Carpentry
+    1.  the date and moderator of the discussion session they have taken part in
+        *   we currently store both values in one column
+	*   we do not record both a Software Carpentry and a Data Carpentry discussion session (though we should)
+    1.  the date on which they passed their Software Carpentry lesson demo (if any)
+	*   this date becomes the award date of the PDF certificate
+        *   we do not currently record the name of the person who was their instructor, or the name of the person who did their final test
+    1.  the date on which they passed their Data Carpentry lesson demo (if any)
+        *   the same caveats apply
+1.  Mentors offer lesson discussion sessions,
+    and trainees sign up for them,
+    on an Etherpad.
+1.  Trainers offer checkout sessions on a second Etherpad.
+    Trainees sign up for sessions on that Etherpad,
+    which we also presently use for taking notes during their sessions.
+1.  Discussion and checkout sessions are marked in Software Carpentry's community calendar
+    (a Google calendar).
+
+## Issues
+
+1.  As of Jan 2016,
+    the form for individuals to apply to take part in training has been disabled
+    while we try to clear our backlog (and come up with a better process).
+1.  We used a Google web form and a spreadsheet for group applications and sorted manually.
```

Comment:
Ah ok, if we're doing wishlists then I'm happy to jump in:

 - **UI Layout**: Of all the many things I spend my time on, this is the one I actually get paid for; I'm happy to jump on board to wrangle your front end. In any case, keeping it simple is crucial both for ease of cross-browser maintenance and usability. Don't bother with columnar layout, that'll murder your usability and not translate to mobile; if you need a visual representation, a bootstrap progress bar (you are bootstrapping this, right?) will do nicely. I also wouldn't overcomplicate things with comments for every single transition; they are all pass/fail arbitrated by a single person, a single freeform text box for the whole record will do.

 - **Registration:** around line 11, you mention that applicants can sign up for instructor training somewhere on our website, and somewhere else you mention that there's a place where instructors can enter personal info on AMY. Are these all the same thing? If not, that's an opportunity to kill off a moving part or two. Ideally, I would like AMY to have an 'instructor login' where new instructors can create an instructor profile themselves, so we can give them an option to self-identify things like gender and race - we need to be doing our due diligence that success in instructor training isn't accidentally biased on these lines, but we're torries-killing-the-census dead in the water until we start collecting this info. Also, new instructor profiles can easily be rounded up for instructor training cohorts from here.

 - **Group views:** yes! we will need something that lets us view stubs for all instructors that match a search result. This view can be really simple (name / airport / has badge for each match, one match per row, maybe a button to send a CSV list of email addresses for the current search result to the clipboard) - what's more substantial is the actual search widget that lets us make these queries; that's one place where I anticipate bells and whistles will actually pay off, as they'll lets us be more expressive in our queries. Or, a simple search, and the opportunity for power users to enter an SQL-syntax query. 

One other thing on the topic of grouping that has been discussed lately is a deputization model to shepherd people through the post-instructor-training-workshop phase (kind of like what Belinda does for the Australian community and what Tiffany and I do in Vancouver), and that these cohorts might be related by shared interest or professional affiliation rather than just geography; profile information that reflects this info will need to be present to form useful groupings, though there is still conversation to be had on what exactly this will look like - we could assign volunteer deputies to new instructors (to be coordinated with Mentorship, but this requires some bandwidth), or register relevant professional affiliations for instructors in their profile (low bandwidth for us, but probably low-effectiveness as well - will this be a freeform field? how will we munge it to match 'APS', 'Amer. Phys. Soc.', and 'American Physical Sciety'? Or will it be a predefined list? Pulled from where?)

Hope this helps - TLDR: keep it simple.